---
title: "Mitotic checkpoint gene expression is tuned by codon usage bias"
collection: publications
category: manuscripts
permalink: /publication/2022-07-11-mitotic-checkpoint/
excerpt: 'Excerpt here.'
date: 2022-07-11
venue: 'EMBO Journal'
paperurl: /MortonC78483.github.io/files/embo_esposito_2022.pdf
citation: 'Esposito, E., Weidemann, D. E., Rogers, J. M., Morton, C. M., Baybay, E. K., Chen, J., & Hauf, S. (2022). &quot;Mitotic checkpoint gene expression is tuned by codon usage bias&quot;. <i>The EMBO journal</i>, 41(15), e107896.'
---

The mitotic checkpoint (also called spindle assembly checkpoint, SAC) is a signaling pathway that safeguards proper chromosome segregation. Correct functioning of the SAC depends on adequate protein concentrations and appropriate stoichiometries between SAC proteins. Yet very little is known about the regulation of SAC gene expression. Here, we show in the fission yeast <i>Schizosaccharomyces pombe</i> that a combination of short mRNA half-lives and long protein half-lives supports stable SAC protein levels. For the SAC genes <i>mad2<sup>+</sup></i> and <i>mad3<sup>+</sup></i> , their short mRNA half-lives are caused, in part, by a high frequency of nonoptimal codons. In contrast, <i>mad1<sup>+</sup></i> mRNA has a short half-life despite a higher frequency of optimal codons, and despite the lack of known RNA-destabilizing motifs. Hence, different SAC genes employ different strategies of expression. We further show that Mad1 homodimers form co-translationally, which may necessitate a certain codon usage pattern. Taken together, we propose that the codon usage of SAC genes is fine-tuned to ensure proper SAC function. Our work shines light on gene expression features that promote spindle assembly checkpoint function and suggests that synonymous mutations may weaken the checkpoint.
